packages/tailwind/tailwind-base.cjs

```diff
@@ -122,6 +255,24 @@ module.exports = (options = {}) => {
           '.h4': {
             [h4]: {},
           },
+          '.h5': {
+            [h5]: {},
+          },
+          '.h6': {
+            [h6]: {},
+          },
+          '.body': {
```

Comment:
Hva tenker vi om en klasse som bare heter `body`? Blir det litt for generisk?

Reply:
Hvis vi skal ha et prefiks så tenker jeg kanskje at `body` kan funke fint, f.eks. `text-body`? Hvis ikke, så ja da er kanskje `body` litt forvirrende, siden man fort kan tenke at det har med `<body>`-taggen å gjøre? Selv om det er riktig typografisk terminologi. I så fall kunne man kanskje kalt den for `body-text`?

Reply:
Nytt forslag her er `paragraph`